src/init/pkg_installer.sh

```diff
@@ -2,52 +2,53 @@
 
 # Copyright (C) 2015-2020, Wazuh Inc.
 
-. /etc/ossec-init.conf
+WAZUH_HOME=${1}
+WAZUH_VERSION=${2} 
 
 # Generating Backup
 BDATE=$(date +"%m-%d-%Y_%H-%M-%S")
 
-echo "$(date +"%Y/%m/%d %H:%M:%S") - Generating Backup." > ${DIRECTORY}/logs/upgrade.log
-mkdir -p ${DIRECTORY}/tmp_bkp/${DIRECTORY}/bin
-mkdir -p ${DIRECTORY}/tmp_bkp/${DIRECTORY}/etc
-mkdir -p ${DIRECTORY}/tmp_bkp/etc
+echo "$(date +"%Y/%m/%d %H:%M:%S") - Generating Backup." > ${WAZUH_HOME}/logs/upgrade.log
+mkdir -p ${WAZUH_HOME}/tmp_bkp/${WAZUH_HOME}/bin
+mkdir -p ${WAZUH_HOME}/tmp_bkp/${WAZUH_HOME}/etc
+mkdir -p ${WAZUH_HOME}/tmp_bkp/etc
 
-cp -rp ${DIRECTORY}/bin ${DIRECTORY}/tmp_bkp/${DIRECTORY}
-cp -rp ${DIRECTORY}/etc ${DIRECTORY}/tmp_bkp/${DIRECTORY}
-cp -p /etc/ossec-init.conf ${DIRECTORY}/tmp_bkp/etc
+cp -rp ${WAZUH_HOME}/bin ${WAZUH_HOME}/tmp_bkp/${WAZUH_HOME}
+cp -rp ${WAZUH_HOME}/etc ${WAZUH_HOME}/tmp_bkp/${WAZUH_HOME}
+cp -p /etc/ossec-init.conf ${WAZUH_HOME}/tmp_bkp/etc
```

Comment:
Now that the `/etc/ossec-init.conf` file is deprecated, this line may generate errors if the upgrade script doesn't find it in newest versions.

Reply:
This script is forked from **upgrade.sh** as a WPK requirement so, the error messages will not rise to an open terminal or process. In fact, the useful information is stored in upgrade.log for tracking.
But, it's true that this line will generate confusion as **ossec-init.conf** will not be necessarily installed. 
Modified to now check the existence of the file before **cp**.